### DISCUSSION TITLE
Cannot pool fitted mixed effects models

### DISCUSSION THREAD

**Discussion by studerus** [amices/mice#362] [Impute, analyse and pool]
Pooling multiple fitted mixed effects models results in the following error message:
`Error: No tidy method for objects of class lme`

Here's a minimal example:
```
library(mice)
library(nlme)
dat <- nhanes
dat$id <- 1:5
imp <- mice(dat, maxit = 2, m = 2)
fit <- with(imp, exp = lme(bmi ~ age, random = ~1|id))
pool(fit)
```

I have already installed the developmental versions of mice and broom from github, but it didn't solve the problem.

**Comment by stefvanbuuren** [comment]
Thanks. Please add

```
library(broom.mixed)
```

and then it should work.

**Comment by stefvanbuuren** [comment]
Patrick D'Onofrio alerted me that `library(broom.mixed)` also solved error an `Error in as.data.frame.default(data): cannot coerce class '"mids"' to a data.frame`.
